package.json

```diff
@@ -19,5 +19,8 @@
   "bugs": {
     "url": "https://github.com/mvdicarlo/postybirb-plus/issues"
   },
-  "homepage": "https://github.com/mvdicarlo/postybirb-plus#readme"
+  "homepage": "https://github.com/mvdicarlo/postybirb-plus#readme",
+  "dependencies": {
+    "yarn": "^1.22.4"
```

Comment:
Not sure about including this as a dependency. Only because I am unsure how it will interact with the appveyor build but I guess I can figure that out if it causes problems.